slack/events.w

```diff
@@ -0,0 +1,23 @@
+pub struct SlackEvent {
+  type: str;
+}
+
+pub struct CallBackEvent {
+  user: str;
+  type: str;
+  ts: str;
+  team: str;
+  channel: str;
+  event_ts: str;
+  bot_id: str?;
+  app_id: str?;
+}
+
+pub struct MessageCallBackEvent extends CallBackEvent {
```

Comment:
Super nitpick (I'm guessing "callback" is meant in this sense? https://en.wikipedia.org/wiki/Callback_(computer_programming))
```suggestion
pub struct MessageCallbackEvent extends CallbackEvent {
```

Reply:
Yea you are right here.